container/docker/handler.go

```diff
@@ -217,7 +216,6 @@ func newDockerContainerHandler(
 		Namespace: DockerNamespace,
 	}
 	handler.image = ctnr.Config.Image
-	handler.networkMode = ctnr.HostConfig.NetworkMode
```

Comment:
why was this removed?

Reply:
This is because the only user of `h.networkMode` was `func (h *dockerContainerHandler) needNet() bool`.

That method is now gone, and so the field is unused.